recipes/os_command_py/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "os_command_py" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5bfec656e39ccc2a2c3c76b3d40bf7437d4d597b603d60f4f8d11cd123f0a889
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+  skip: true  # [win]
```

This looks like it can be `noarch: python`. It's okay for Windows to fail after this is applied
```suggestion
  noarch: python
```